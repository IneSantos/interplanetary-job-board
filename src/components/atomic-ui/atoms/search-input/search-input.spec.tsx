import React from "react";
import { render } from '@testing-library/react';
import SearchInput from "./search-input";

describe("SearchInput atom tests", () => {
  test(`SearchInput component render`, () => {
    const { container } = render(
      <SearchInput input={"this is an input"} handleSearch={jest.fn()}/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
