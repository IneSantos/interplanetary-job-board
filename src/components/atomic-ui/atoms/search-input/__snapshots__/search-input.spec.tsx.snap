// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput atom tests SearchInput component render 1`] = `
<input
  class="search-term"
  placeholder="What are you looking for?"
  type="text"
  value="this is an input"
/>
`;
